jest.config.base.js

```diff
@@ -37,7 +37,31 @@ module.exports = {
         '!<rootDir>/**/jump-consistent-hash.*',
         '!<rootDir>/**/guid-generator.*',
     ],
-    reporters: ['default', ['jest-junit', { outputDirectory: '<rootDir>/test-results/unit', outputName: 'junit.xml' }]],
+    reporters: [
+        'default',
+        [
+            'jest-junit',
+            {
+                outputDirectory: '<rootDir>/test-results/unit',
+                outputName: 'junit.xml',
+            },
+        ],
+        [
+            'jest-html-reporter',
+            {
+                pageTitle: 'html report',
+                outputPath: './test-results/unit/html-reporter/report.html',
+            },
+        ],
+        // [
+        //     'jest-slow-test-reporter',
```

Comment:
Why do we need to keep this?

Reply:
we shouldnt enable this as it increases test run time. But, if we want to find slow running tests, we just need to uncomment this & run tests. to fix this issue (https://github.com/microsoft/accessibility-insights-service/issues/601)